Sources/Subs.php

```diff
@@ -824,6 +826,13 @@ function timeformat($log_time, $show_today = true, $offset_type = false, $proces
 		{
 			foreach($strftimeFormatSubstitutions as $format => $substitution)
 			{
+				// Avoid a crashing bug with PHP 7 on certain versions of Windows
+				if ($context['server']['is_windows'] && in_array($format, $unsupportedFormatsWindows))
```

Comment:
Should be `in_array($format, array('z', 'Z'))`

Reply:
In my eyes is worst to read and hard to maintaine,
reasone why i keep to define the array in front of the function.

Reply:
Further evidence that I should go to bed now.